scripts/install

```diff
@@ -93,8 +93,17 @@ ynh_setup_source "$final_path"
 # INSTALL SEARX IN A VIRTUALENV
 #=================================================
 
-virtualenv --system-site-packages "$final_path"
-bash -c "source $final_path/bin/activate && pip install -U setuptools && pip install --requirement $final_path/requirements-ynh.txt"
+virtualenv "$final_path"
+#run source in a 'sub shell'
+(
+	set +o nounset
+	source "$final_path/bin/activate"
+	set -o nounset
+	pip install --upgrade pip
```

Comment:
pip has been removed from here to prevent conflict https://github.com/YunoHost-Apps/searx_ynh/pull/31